lib/rightscript_sync.rb

```diff
@@ -69,45 +69,66 @@ def set_metadata(*files)
   end
 
   desc "download <file> [-n -i] <script dash name>", "Download RightScript to file"
-  method_option :id, 
+  method_option :id,
     :desc => "RightScript ID number to download",
     :aliases => "-i",
     :type => :numeric
-  method_option :name, 
+  method_option :name,
     :desc => "RightScript Name to download",
     :aliases => "-n"
   def download(file)
```

Comment:
Looking at the code, if `options[:id]` or `options[:name]` is not provided, `file` is then treated as the location to read filenames and download based on those names.  If the options are provided, `file` is now the location to download the script to.  This overloads the `file` variable. We should either have another option (`options[:directory]`), or another method.